tests/syscall/mount_syscall.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"golang.org/x/sys/unix"
+)
+
+func do_bind_mount(s, t string) {
+
+	err := unix.Mount(s, t, "", unix.MS_BIND, "")
+	if err != nil {
+		fmt.Printf("bind-mount error received %v:", err)
+	}
```

Comment:
We should return an error here, and have main() exit with an error code (so bats can catch the failures).

Reply:
yep, done